components/match2/wikis/smite/match_group_input_custom.lua

```diff
@@ -129,7 +130,7 @@ function CustomMatchGroupInput.getResultTypeAndWinner(data, indexedScores)
 	-- Map or Match is marked as finished.
 	-- Calculate and set winner, resulttype, placements and walkover (if applicable for the outcome)
 	elseif Logic.readBool(data.finished) then
-		if MatchGroupInput.isDraw(indexedScores) then
+		if CustomMatchGroupInput.placementCheckDraw(indexedScores) then
```

Comment:
Why changing from a standardized function to a custom one? A function that doesn't seem to exist?

Reply:
It did exist, along with several other custom methods that lined up with commons versions. Removed those and it should be commons based now